Traceback (most recent call last):
  File "train.py", line 186, in <module>
    main(arguments)
  File "train.py", line 145, in main
    X, y = load_data(args)
  File "/home/mburu/Master_Thesis/master-thesis-da/DNN_Trial/utils/load_data.py", line 183, in load_data
    df = pd.read_csv('/home/mburu/Master_Thesis/master-thesis-da/datasets/42688-Brazilian_houses/raw_data.csv') #CLUSTER
UnboundLocalError: local variable 'pd' referenced before assignment
srun: error: stud-000: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "train.py", line 186, in <module>
    main(arguments)
  File "train.py", line 145, in main
    X, y = load_data(args)
  File "/home/mburu/Master_Thesis/master-thesis-da/DNN_Trial/utils/load_data.py", line 207, in load_data
    df = pd.read_csv('/home/mburu/Master_Thesis/master-thesis-da/datasets/42731-house_sales/raw_data.csv') #CLUSTER
UnboundLocalError: local variable 'pd' referenced before assignment
srun: error: stud-000: task 0: Exited with exit code 1
